tools/depends/target/Makefile

```diff
@@ -97,6 +97,7 @@ pythonmodule-pil: $(ZLIB) libjpeg-turbo libpng freetype2 python26
 libsdl2: $(LINUX_SYSTEM_LIBS)
 libxslt: libgcrypt
 ffmpeg: $(ICONV) $(ZLIB) bzip2 libvorbis $(FFMPEG_DEPENDS)
+libcec: platform
```

Comment:
No problem, will do. If you have time, please review the PRs. Knowing myself, I'm gonna forget something totally obvious/stuppid.